Dockerfile

```diff
@@ -1,7 +1,18 @@
-FROM bentoml/model-server:0.11.0-py37
+FROM bentoml/model-server:0.11.0-py38
 MAINTAINER ersilia
 
 RUN pip install rdkit
+RUN pip install git+https://github.com/bp-kelley/descriptastorus
+RUN pip install tqdm>=4.62.2
+RUN pip install typed-argument-parser==1.6.1
+RUN pip install scikit-learn
+RUN pip install torch
+RUN pip install pandas
+RUN pip install tensorboardX==2.0
```

Comment:
Same as above. Tensorboard is a visualization tool, do we really need it here? https://www.tensorflow.org/tensorboard